akka-sample-cluster-scala/src/main/scala/sample/cluster/transformation/TransformationBackend.scala

```diff
@@ -24,7 +22,7 @@ class TransformationBackend extends Actor {
   override def preStart(): Unit = cluster.subscribe(self, classOf[MemberUp])
   override def postStop(): Unit = cluster.unsubscribe(self)
 
-  def receive = {
+  def receive: PartialFunction[Any, Unit] = {
```

Comment:
we often don't include that for `receive`, and then would be better to use `receive: Receive`, which is a type alias

Reply:
Sure @patriknw , will fix this one.